.github/workflows/update-deps.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
       - name: update dependencies url
         run: |
           ./.github/bin/update-lima-dep.sh -d ${{ secrets.DEPENDENCY_BUCKET_NAME }} -a {{ secrets.ARTIFACT_BUCKET_NAME }}
+          ./.github/bin/update-rootfs.sh -d ${{ secrets.DEPENDENCY_BUCKET_NAME }}
```

Comment:
Can we have a different bucket/sub-folder for the new artifact?

Reply:
sure, or how about under respective ${ARCH} folder as lima/qemu is today? 
https://github.com/runfinch/finch/blob/81d8b20d0253016bcfd49f72d763ec3aaa93b7af/Makefile#L38

Reply:
I think a folder at the same level with aarch64 and amd64 maybe better so that each type of the artifact won't be stored in the same folder which will be cleaner and reduce the risk for artifact remove case for a dir.

Reply:
By the way, I think we may only need this change in the Core repo

Reply:
> By the way, I think we may only need this change in the Core repo

Yea thanks for pointing that out... There are a couple places where we can deduplicate code / actions between finch and finch-core. Opened https://github.com/runfinch/finch/issues/508 to track